compiler/src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -318,6 +325,12 @@ class PostTyper extends MacroTransform with IdentityDenotTransformer { thisPhase
           throw ex
       }
 
+    /** Transforms the rhs tree into a its default tree if it is in an `erased` val/def.
```

Comment:
Shouldn't this be done after pickling so that "find all references" in the IDE does not miss references in erased rhs?

Reply:
Yes, good point. It used to be done in Typer, but you are right wrt IDEs. We should eliminate implicit arguments before pickling though, since they can get very large, and we only need the type, but not the term for IDEs.